external-crates/move/crates/move-core-types/src/resolver.rs

```diff
@@ -26,11 +26,17 @@ pub struct TypeOrigin {
 /// storage IDs.
 #[derive(Debug, Clone)]
 pub struct SerializedPackage {
-    pub modules: Vec<Vec<u8>>,
```

Comment:
I am a little dubious about this change: we have previously discussed `init` relying on this order (which is currently sorted based on dependencies), and we need to consider how `init` and `init_with_args` will look with these changes.